spacewalk/config/spacewalk-config.changes

```diff
@@ -1,3 +1,4 @@
+- remove superfluous 'apache' entries from sudoers configuration
```

Comment:
I think you created a bug for it. Please add the bugnumber.

Reply:
Ah, of course you are right. Fixed.